dspace/config/registries/bitstream-formats.xml

```diff
@@ -201,7 +201,7 @@
 
   <bitstream-type>
     <mimetype>image/png</mimetype>
-    <short_description>image/png</short_description>
+    <short_description>PNG</short_description>
```

Comment:
@alanorth : While I agree this short_description has been wrong for a long time... I'm a little worried about making this change _without_ some sort of migration to automatically update everyone's database entries to use "PNG" instead of "image/png".

Without the migration in place, I'm worried that the Media Filter changes you've made in `dspace.cfg` will no longer work for sites which still have "image/png" instead of "PNG".

So, I think we should either..
1. Create a database migration to migrate all sites from "image/png" to "PNG" in the bitstream format table (where it currently exists)
2. OR, we should undo this change and simply leave it as "image/png" for now....even though it's slightly wrong.

Reply:
Good point, Tim! I think it's worth fixing the have this be correct instead of carrying around legacy typos forever. I will look into creating a database migration to update `image/png` for existing users since the bitstream format registry is only run on new installs.

Reply:
I've added a SQL migration for PostgreSQL, Oracle, and H2, though I have only tested it on PostgreSQL.